airbyte-integrations/connectors/destination-bigquery/src/main/java/io/airbyte/integrations/destination/bigquery/uploader/BigQueryUploaderFactory.java

```diff
@@ -115,10 +114,12 @@ private static BigQueryDirectUploader getBigQueryDirectUploader(
     }
 
     // this this optional value. If not set - use default client's value (15MiG)
-    final Integer bigQueryClientChunkSizeFomConfig =
-        BigQueryUtils.getBigQueryClientChunkSize(config);
-    if (bigQueryClientChunkSizeFomConfig != null) {
-      writer.setChunkSize(bigQueryClientChunkSizeFomConfig);
+    if (bigQueryClientChunkSize != null) {
+      if (bigQueryClientChunkSize <= 0) {
+        LOGGER.error("BigQuery client Chunk (buffer) size must be a positive number (MB), but was:" + bigQueryClientChunkSize);
+        throw new IllegalArgumentException("BigQuery client Chunk (buffer) size must be a positive number (MB)");
+      }
+      writer.setChunkSize(bigQueryClientChunkSize * BigQueryConsts.MiB);
```

Comment:
This logic existed before in BigQueryUtils when extracting the value from JsonNode